tests/network/vmi_subdomain.go

```diff
@@ -50,9 +51,13 @@ var _ = SIGDescribe("Subdomain", func() {
 		tests.BeforeTestCleanup()
 	})
 
+	const (
+		subdomain     = "testsubdomain"
+		k8sNameserver = "10.96.0.10"
```

Comment:
It is the default Ipv4 k8s cluster dns name server

Reply:
We do not always control the cluster on which these tests run against.
I do not think you can assume it.

Perhaps you can read it from some other pod or just invent one with a link-local address.

Reply:
maybe instead of this, we can change the test then to only check the expected resolv.conf instead `hostname -f` ?
(after all this what should be checked by this test, the rest might be additionals)
and then we can use this ip but it won't actually be used.
This what happens as well on other test that uses `DNSPolicy` None
and it will keep it simple

Reply:
This is an e2e test, we are interested to check all the path if possible. I am sure there is a way to read this value using client-go or alike. I think we do have code that checks services already, no?

Reply:
I just verified on Downstream cluster, they don't have the same service as we have,
so if we read it, we better read it from a pod, for example virt-handler that always should exists